funcbench/main.go

```diff
@@ -332,7 +343,12 @@ func (c *commander) exec(command ...string) (string, error) {
 	if err := cmd.Run(); err != nil {
 		out := b.String()
 		if c.verbose {
-			out = ""
+			_, ok := c.env.(*GitHub)
+			if ok {
+				out = fmt.Sprintf("<details><summary>Error:</summary>\n```\n%s\n```\n</details>", out)
```

Comment:
Why not just:
````

if out != "" {
     out = fmt.Sprintf("<details><summary>Error:</summary>\n```\n%s\n```\n</details>", out)
}
```

Reply:
Since it'll be slightly complicated to have loki for funcbench since it runs on a different cluster( #385 ) I thought it'll be nice to have the command line error on the github comment aswell.

Previously, it just set `out` to an empty string when run in verbose mode. (we run with verbose flag on when running funcbench on Github mode)

Reply:
sure I understand the logic, but `_, ok := c.env.(*GitHub)` looks like a design flow.

Reply:
Yes, that seemed a bit odd. I updated this PR to include the error output in verbose mode aswell. It doesn't add a lot of noise so it should be fine.

Plus now it will always then give us the error message(run in verbose flag set or not) when running in GitHub mode.

Will test this and post example comments.